### PR TITLE
remaining call center indicator tests running on SQL

### PR DESCRIPTION
**Pull request by snopoke**
@orangejenny 

cc @NoahCarnahan @czue 

review by commit :fish:

### CODE REVIEW

File: corehq/util/test_utils.py [hunk 1]
```diff
@@ -232,6 +234,23 @@ def inner(*args, **kwargs):
     return inner
 
 
+class OverridableSettingsTestMixin(object):
```
Comment (snopoke) [inline corehq/util/test_utils.py]: @dimagi/team-commcare-hq FYI if you want to run tests that use `setUpClass` / `tearDownClass` but also want to override settings.

File: corehq/util/test_utils.py [hunk 2]
```diff
@@ -232,6 +234,23 @@ def inner(*args, **kwargs):
...
+    """Backport of core Django functionality to 1.7. Can be removed
+    once Django >= 1.8
+    https://github.com/django/django/commit/d89f56dc4d03f6bf6602536b8b62602ec0d46d2f"""
+    @classmethod
+    def setUpClass(cls):
+        if cls._overridden_settings:
```
Comment (millerdev) [inline corehq/util/test_utils.py]: What do you think of adding a usage example in the doc string. Something like:
```
Usage:

  @override_settings(A_SETTING=True)
  class SomeTests(TestCase, OverridableSettingsTestMixin):

      @classmethod
      def setUpClass(cls):
          super(SomeTests, cls).setUpClass()
          # settings.A_SETTING is True here

      @classmethod
      def tearDownClass(cls):
          # teardown stuff
          # don't forget to call super to undo override_settings
          super(SomeTests, cls).tearDownClass()
```
It's not immediately obvious that this is connected to the `override_settings` class decorator and that you need to call `super` when overriding `setUpClass`/`tearDownClass`.

Comment (snopoke) [inline corehq/util/test_utils.py]: 5496a7a

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by millerdev** [comment]
lintreview ran but travis did not. Any idea why?

**Comment by millerdev** [comment]
nm. I see `[ci skip]` caused travis to be skipped.

**Pull request merged**